File: Niveau 2/Découverte des tableaux/Course à trois jambes/main.cpp
#include <iostream>
#include <algorithm>
int main()
{
  int nbParticipants{0};
  std::cin >> nbParticipants;
  int choix [3000] = {0};
  for (int i{0}; i < nbParticipants; i++)
  {
    std::cin >> choix [i];
  }
  std::sort(choix, choix + nbParticipants);
  for (int i{0}; i < nbParticipants / 2; i++)
  {
    std::cout << choix [i] << " " << choix [nbParticipants - i - 1] << std::endl;
  }
  return 0;
}

File: Niveau 2/Découverte des tableaux/Course à trois jambes/main.py
nbParticipants = int(input())
nombre = [0] * nbParticipants
for loop in range(nbParticipants):
    nombre[loop] = int(input())
nombre.sort()
for loop in range(nbParticipants // 2):
    print("{} {}".format(nombre[loop], nombre[nbParticipants - loop - 1]))
